File: Iniciante/URI 1005.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val a: Double = leitor.nextDouble();
    val b: Double = leitor.nextDouble();
    val media: Double = ((3.5 * a) + (7.5 * b)) / 11;

    println(String.format("MEDIA = %.5f", media));
}

File: Iniciante/URI 1005.scala
import java.util.Scanner

object Main {
    
    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val a = leitor.nextDouble()
        val b = leitor.nextDouble()
        val media = ((3.5 * a) + (7.5 * b)) / 11
        printf("MEDIA = %.5f\n", media)
    }
    
}